src/main/java/com/elytradev/concrete/inventory/gui/widget/WSlot.java

```diff
@@ -110,14 +101,12 @@ public int getHeight() {
 	
 	@Override
 	public void createPeers(ConcreteContainer c) {
-		peers.clear();
```

Comment:
The reason we *always, always* clear peers on createPeers is that double-calls of this method are quite common! So no, definitely leave that in.